draft-ietf-acme-dns-account-01.mkd

```diff
@@ -172,6 +172,37 @@ If all the above verifications succeed, then the validation is successful. If no
 
 The client SHOULD de-provision the resource record(s) provisioned for this challenge once the challenge is complete, i.e., once the "status" field of the challenge has the value "valid" or "invalid".
 
+## Errors
+
+The server SHOULD follow the guidelines set in {{!RFC8555, Section 6.7}} for error conditions that occur during challenge validation.
+
+If the server is unable to find a TXT record for the validation domain name, it MUST include the account URL it used to construct the validation domain name in a subproblem of the problem document. The subproblem MUST have a type of `urn:ietf:params:acme:error:account_url`. It MUST include an identifier field with a type of `account_url` and value of the account URL.
+
+```json
+HTTP/1.1 403 Forbidden
+Content-Type: application/problem+json
+Link: <https://example.com/acme/directory>;rel="index"
+
+{
+    "type": "urn:ietf:params:acme:error:dns",
+    "detail": "Unable to validate challenge",
+    "subproblems": [
+        {
+            "type": "urn:ietf:params:acme:error:account_url",
```

Comment:
This shouldn't be that hard to add:

https://www.iana.org/assignments/acme/acme.xhtml#acme-error-types